atf_eregs/static/regulations/css/less/typography.less

```diff
@@ -392,6 +392,10 @@ The `<ol>` tag needs to have the appropriate list type applied
 ```
 */
 
+.main-content {
+    overflow-x:scroll;
+}
```

Comment:
What does this look like for very long paragraphs? Does the text wrap or does the page scroll?